README.md

```diff
@@ -32,15 +32,15 @@ First install the neo-sophia code on your local machine before proceeding to the
 
 If the tests pass you are ready to run the code in one of the Episodes.
 
-Questions? Just ask in our [Discord Community](#).
+Questions? Just ask in our [Discord Community](https://discord.gg/tzjEEjyg).
 
 # Episodes
 ## Episode 3 - Intro to Unified Natural Language Query
 What will be the AI "killer app" in the enterprise? Our bet is Unified Natural Language Query (NQL). It gives executives and business leaders the ability to get insights from data by asking "natural" questions, similar to how you currently use ChatGPT. In this Episode we describe the business problem and show the extensible power of a simple example of SQL generation supplemented with the reasoning power of an LLM like GPT-4.
 
-Videos
-- [Unified Natural Language Query is the enerprise AI "killer app"](#)
-- [SQL generation and interpretation with LLMs](#)
+Videos (coming soon!)
+- [Unified Natural Language Query is the enerprise AI "killer app"]
+- [SQL generation and interpretation with LLMs]
```

Comment:
With the parentheses and `#`, these will show up as links that don't go anywhere. Removing the parentheses they are no longer valid markdown links and look like this: 
![image](https://github.com/prolego-team/neo-sophia/assets/10505370/4f14cf28-abae-4215-af3b-df2c82615736)